pkg/chartrenderer/default.go

```diff
@@ -187,3 +187,13 @@ func (c *RenderedChart) FileContent(filename string) string {
 	}
 	return ""
 }
+
+// AsSecretData returns all rendered manifests that is capable for used as data of a secret
+func (c *RenderedChart) AsSecretData() map[string][]byte {
+	data := make(map[string][]byte, len(c.Files()))
+	for fileName, fileContent := range c.Files() {
+		key := strings.Replace(fileName, "/", "_", -1)
```

Comment:
WDYT, about using `strings.ReplaceAll`. It should yield the same results, but is probably nicer to read :)
```suggestion
		key := strings.ReplaceAll(fileName, "/", "_")
```